docs/BrandColors.stories.tsx

```diff
@@ -18,7 +21,26 @@ export default meta;
 
 type Story = StoryObj<typeof ColorSwatchGroup>;
 
-export const DefaultStory: Story = {
+export const Figma: Story = {
   render: () => <ColorSwatchGroup swatchData={tokens.global.brandColors} />,
-  name: 'Default',
+};
+
+export const CSS: Story = {
+  render: () => (
+    <div
+      style={{
+        display: 'grid',
+        gap: '16px',
+        gridTemplateColumns: 'repeat(auto-fill, 300px)',
+      }}
+    >
+      {Object.values(brandColors).map(({ color, name }) => (
+        <ColorSwatch key={name} color={color} name={name} />
+      ))}
+    </div>
+  ),
+};
+
+export const JS: Story = {
+  render: () => <h1>Coming soon</h1>,
 };
```

Comment:
Will update once we add the `brandColor` object https://github.com/MetaMask/design-tokens/pull/585

Reply:
Updated